src/components/Banner.js

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useState } from "react";
+import "./Banner.css";
+import { getUserInfo } from "../api/api";
+import avatarImg from "../assets/png/avatar.png";
+
+function Banner() {
+  const [userData, setItems] = useState([]);
```

Comment:
useState 훅의 리턴값을 사용할 때 네이밍 컨벤션은 꼭 지켜주는게 좋아요~
```suggestion
  const [userData, setUserData] = useState([]);
```

Reply:
userData라는 이름으로 배열 데이터 형식이라는 걸 추측하기는 조금 어려울 것 같은데 네이밍을 바꿔보면 좋을 것 같아요~ 여기서도 -Data와 같은 표현은 변수의 목적 자체가 어떤 값, 데이터를 담는 것이기때문에 필요없는 네이밍일 것 같네요!

Reply:
사용하는쪽에서 `userData.name`라고 사용하는걸 보면 이 상태값에는 배열 형식의 데이터가 담기는게 아닌 것 같네요! 초기값 선택에 문제가 있었던 것 같아요~